lib/Client.php

```diff
@@ -6,7 +6,7 @@ class Client
 {
     private const MONITOR_TYPES = ['job', 'event', 'synthetic'];
     private const BASE_CONFIG_KEYS = ['apiKey', 'apiVersion', 'environment'];
-  
+
```

Comment:
I just got `php-cs-fixer` working (seems to be the most common PHP formatter, used by Stripe), so there's a lot of spacing/formatting fixed.